script/input/1/base-production.json

```diff
@@ -0,0 +1,9 @@
+{
+    "admin": "0xF93B7122450A50AF3e5A76E1d546e95Ac1d0F579",
+    "almProxy": "0x2917956eFF0B5eaF030abDB4EF4296DF775009cA",
+    "cctpTokenMessenger": "0x1682Ae6375C4E4A97e4B583BC394c861A46D8962",
+    "psm": "0x1601843c5E9bC251A3272907010AFa41Fa18347E",
+    "rateLimits": "0x983eC82E45C61a42FDDA7B3c43B8C767004c8A74",
+    "safe": "0xaB959A6F88b8D966c44a7cDC2049Ba9669EBf047",
```

Comment:
Production safe is `0x8a25A24EDE9482C4Fc0738F99611BE58F1c839AB`

Reply:
Updated